components/net/bluetooth_thread.rs

```diff
@@ -149,6 +150,12 @@ impl BluetoothManager {
                 BluetoothMethodMsg::GetPrimaryServices(device_id, uuid, sender) => {
                     self.get_primary_services(device_id, uuid, sender)
                 },
+                BluetoothMethodMsg::GetIncludedService(device_id, primary_id, uuid, sender) => {
```

Comment:
Maybe ``` primary_uuid ``` instead of ``` primary_id ``` ? I think ``` the primary_id ``` is a little confusing, because e.g. in the ``` get_characteristics ``` we use a ``` service_id ``` attribute which is an object path, not a uuid.